# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-11-17 22:06+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: auth/config/ticket.py:38
msgid "项目申请结果数据"
msgstr ""

#: auth/config/ticket.py:42
msgid "资源组授权申请"
msgstr ""

#: auth/config/ticket.py:46
msgid "申请角色"
msgstr ""

#: auth/config/ticket.py:50
msgid "授权码申请权限"
msgstr ""

#: auth/config/ticket.py:54
msgid "离线补录申请"
msgstr ""

#: auth/config/ticket.py:58
msgid "TDM原始数据接入申请"
msgstr ""

#: auth/config/ticket.py:62
msgid "资源组创建申请"
msgstr ""

#: auth/config/ticket.py:67
msgid "资源组扩容申请"
msgstr ""

#: auth/constants.py:20 auth/constants.py:28
msgid "已同意"
msgstr ""

#: auth/constants.py:21 auth/constants.py:29
msgid "已驳回"
msgstr ""

#: auth/constants.py:22
msgid "待处理"
msgstr ""

#: auth/constants.py:23 auth/constants.py:30
msgid "处理中"
msgstr ""

#: auth/constants.py:24 auth/constants.py:31
msgid "已终止"
msgstr ""

#: auth/constants.py:44
msgid "有效"
msgstr ""

#: auth/constants.py:45
msgid "无效"
msgstr ""

#: auth/constants.py:46
msgid "申请中"
msgstr ""

#: auth/constants.py:64
msgid "用户"
msgstr ""

#: auth/constants.py:65
msgid "APP"
msgstr ""

#: auth/constants.py:66 auth/constants.py:68
msgid "项目"
msgstr ""

#: auth/constants.py:67
msgid "授权码"
msgstr ""

#: auth/constants.py:69
msgid "原始数据"
msgstr ""

#: auth/constants.py:70
msgid "业务"
msgstr ""

#: auth/constants.py:84
msgid "公开"
msgstr ""

#: auth/constants.py:85
msgid "接入后平台所有用户均可使用此数"
msgstr ""

#: auth/constants.py:91
msgid "业务私有"
msgstr ""

#: auth/constants.py:92
msgid "由业务负责人直接接入至平台，业务人员可见，平台用户均可申请此数据"
msgstr ""

#: auth/constants.py:98
msgid "业务机密"
msgstr ""

#: auth/constants.py:99
msgid "由业务负责人接入平台，业务成员不可见，数据申请方需要leader级别"
msgstr ""

#: auth/constants.py:105
msgid "业务绝密"
msgstr ""

#: auth/constants.py:106
msgid "由业务负责人接入平台，仅授权人员可见，数据申请方需要总监级别"
msgstr ""

#: auth/core/process.py:87
msgid "目前仅支持 RT 级别的权限申请"
msgstr ""

#: auth/core/process.py:103
msgid "暂不支持敏感级为 {} 的数据申请"
msgstr ""

#: auth/core/process.py:120
msgid "直接通过"
msgstr ""

#: auth/core/process.py:139
msgid "结果表 {} 不存在数据管理员（业务负责人）"
msgstr ""

#: auth/core/process.py:146 auth/core/process.py:192
msgid "数据管理员审批"
msgstr ""

#: auth/core/process.py:169
msgid "结果表 {} 不存在数据管理员（业务总监）"
msgstr ""

#: auth/core/process.py:184
msgid "直属 Leader 审批"
msgstr ""

#: auth/core/process.py:330
msgid "审批人为空，源于角色（{}）不存在成员"
msgstr ""

#: auth/core/ticket_notification.py:115 auth/core/ticket_notification.py:178
msgid "申请人"
msgstr "申请中"

#: auth/core/ticket_notification.py:116
msgid "申请类型"
msgstr ""

#: auth/core/ticket_notification.py:117 auth/core/ticket_notification.py:183
msgid "申请原因"
msgstr ""

#: auth/core/ticket_notification.py:140
#, python-brace-format
msgid ""
"申请人：{0}\n"
"申请类型：{1}\n"
"申请原因：{2}\n"
"申请详情：{3}"
msgstr ""

#: auth/core/ticket_notification.py:156
#, python-brace-format
msgid ""
"您在平台的权限申请单据已审批,审核结果：{0}\n"
"申请人：{1}\n"
"申请类型：{2}\n"
"申请原因：{3}\n"
"申请详情：{4}"
msgstr ""

#: auth/core/ticket_notification.py:179 auth/models/ticket_models.py:87
msgid "单据类型"
msgstr ""

#: auth/core/ticket_notification.py:180
msgid "权限获得者"
msgstr ""

#: auth/core/ticket_notification.py:181
msgid "申请内容"
msgstr ""

#: auth/core/ticket_notification.py:182
msgid "申请范围"
msgstr ""

#: auth/core/ticket_notification.py:225
msgid "【IEG数据平台】您有权限申请单据需要处理"
msgstr ""

#: auth/core/ticket_notification.py:227
msgid "【测试-IEG数据平台】您有权限申请单据需要处理"
msgstr ""

#: auth/core/ticket_notification.py:270
msgid "【IEG数据平台】您在平台的权限申请单据已审批"
msgstr ""

#: auth/core/ticket_notification.py:272
msgid "【测试-IEG数据平台】您在平台的权限申请单据已审批"
msgstr ""

#: auth/core/ticket_objects.py:563 auth/services/role.py:62
msgid "更新部分角色成员失败（{}），请联系管理员"
msgstr ""

#: auth/core/ticket_serializer.py:304
#, python-format
msgid "单据内容缺失 %s"
msgstr ""

#: auth/core/ticket_serializer.py:310
#, python-format
msgid "【AUTH】角色%s非法或未定义"
msgstr ""

#: auth/exceptions.py:17
msgid "资源访问权限不足"
msgstr ""

#: auth/exceptions.py:18
msgid "认证参数缺失"
msgstr ""

#: auth/exceptions.py:18 auth/exceptions.py:19 auth/exceptions.py:24
#: auth/exceptions.py:25
msgid "请查看API请求文档"
msgstr ""

#: auth/exceptions.py:19
msgid "认证不通过，请提供合法的 BKData 认证信息"
msgstr ""

#: auth/exceptions.py:21
msgid "内部模块调用请传递准确的 bk_app_code 和 bk_app_secret"
msgstr ""

#: auth/exceptions.py:22
msgid "传递给的变量需要与 dataapi_settings.py 保持一致"
msgstr ""

#: auth/exceptions.py:23
msgid "数据平台授权码不正确"
msgstr ""

#: auth/exceptions.py:23
msgid "前往平台授权码页面查看授权码"
msgstr ""

#: auth/exceptions.py:24
msgid "未检测到有效的认证信息"
msgstr ""

#: auth/exceptions.py:25
msgid "错误的认证方式"
msgstr ""

#: auth/exceptions.py:26
msgid "ESB 传递的 JWT 字符串解析失败"
msgstr ""

#: auth/exceptions.py:29
msgid "当前用户无单据权限"
msgstr ""

#: auth/exceptions.py:30
msgid "当前单据状态已审批"
msgstr ""

#: auth/exceptions.py:31
msgid "审批范围错误"
msgstr ""

#: auth/exceptions.py:32
msgid "未在可审批阶段"
msgstr ""

#: auth/exceptions.py:33
msgid "无更新权限"
msgstr ""

#: auth/exceptions.py:34
msgid "object类型错误"
msgstr ""

#: auth/exceptions.py:35
msgid "主体校验失败"
msgstr ""

#: auth/exceptions.py:36
msgid "申请范围校验失败"
msgstr ""

#: auth/exceptions.py:37
msgid "已提交过此权限申请，请勿重复提交"
msgstr ""

#: auth/exceptions.py:38
msgid "object不存在"
msgstr ""

#: auth/exceptions.py:39
msgid "action校验错误"
msgstr ""

#: auth/exceptions.py:40
msgid "没有审批人"
msgstr ""

#: auth/exceptions.py:41
msgid "单据类型校验错误"
msgstr ""

#: auth/exceptions.py:42
msgid "重复的权限信息"
msgstr ""

#: auth/exceptions.py:43
msgid "该角色成员不能为空"
msgstr ""

#: auth/exceptions.py:44
msgid "申请的权限对象不存在"
msgstr ""

#: auth/exceptions.py:45
msgid "项目所申请的结果表的区域标签不合法"
msgstr ""

#: auth/exceptions.py:46
msgid "没有匹配的单据类型的审批流程"
msgstr ""

#: auth/exceptions.py:47
msgid "通用单据回调失败"
msgstr ""

#: auth/exceptions.py:48
msgid "单据通知审核回调异常"
msgstr ""

#: auth/exceptions.py:49
msgid "实例不存在"
msgstr ""

#: auth/exceptions.py:50
msgid "不存在与 Meta 映射的对象类型"
msgstr ""

#: auth/exceptions.py:53 auth/permissions.py:77
msgid "授权码不存在"
msgstr ""

#: auth/exceptions.py:54
msgid "授权码未完全通过审批"
msgstr ""

#: auth/exceptions.py:55
msgid "授权码已过期"
msgstr ""

#: auth/exceptions.py:56
msgid "数据范围格式错误"
msgstr ""

#: auth/exceptions.py:57
msgid "不合法的对象范围"
msgstr ""

#: auth/exceptions.py:58
msgid "授权码已被禁止使用"
msgstr ""

#: auth/exceptions.py:61
msgid "暂未绑定TDW账户，请先进行TDW账户绑定"
msgstr ""

#: auth/exceptions.py:62
msgid "没有TDW应用组的权限"
msgstr ""

#: auth/exceptions.py:63
msgid "应用组未绑定到项目上"
msgstr ""

#: auth/exceptions.py:64 auth/exceptions.py:65
msgid "TDW账号或密码错误"
msgstr ""

#: auth/exceptions.py:68
msgid "查询太多资源，拒绝返回内容"
msgstr ""

#: auth/exceptions.py:69
msgid "非法的父级资源类型"
msgstr ""

#: auth/exceptions.py:70
msgid "非法的资源属性"
msgstr ""

#: auth/exceptions.py:71
msgid "资源过滤器无法装换为属性范围"
msgstr ""

#: auth/exceptions.py:74
msgid "BKIAM 不支持通过接口进行属性授权"
msgstr ""

#: auth/exceptions.py:75
msgid "BKIAM 配置限制"
msgstr ""

#: auth/exceptions.py:76
msgid "BKIAM 同步失败"
msgstr ""

#: auth/exceptions.py:79
msgid "基础模型配置不存在PK"
msgstr ""

#: auth/exceptions.py:80
msgid "基础模型实例不存在"
msgstr ""

#: auth/exceptions.py:83
msgid "角色访问权限不足"
msgstr ""

#: auth/exceptions.py:84
msgid "功能还未实现"
msgstr ""

#: auth/exceptions.py:85
msgid "非法APP发起访问，请检查 APP_CODE"
msgstr ""

#: auth/exceptions.py:86
msgid "存在不符合预期的 Model 数据"
msgstr ""

#: auth/exceptions.py:87
msgid "参数校验错误"
msgstr ""

#: auth/exceptions.py:88
msgid "无法连接REDIS服务"
msgstr ""

#: auth/exceptions.py:88
msgid "检查AuthAPI依赖redis服务是否正常"
msgstr ""

#: auth/exceptions.py:89
msgid "ObjectClass 类型定义缺少对主键定义"
msgstr ""

#: auth/exceptions.py:90
msgid "更新角色成员列表错误"
msgstr ""

#: auth/handlers/resource/attributes.py:34 auth/models/auth_models.py:687
#: auth/models/outer_models.py:577 auth/models/outer_models.py:606
#: auth/models/outer_models.py:633 auth/views/auth_serializers.py:277
msgid "敏感度"
msgstr ""

#: auth/handlers/resource/attributes.py:41 auth/models/auth_models.py:162
#: auth/models/outer_models.py:238
msgid "业务ID"
msgstr ""

#: auth/handlers/resource/attributes.py:45
msgid "处理类型"
msgstr ""

#: auth/models/auth_models.py:30
msgid "正常"
msgstr ""

#: auth/models/auth_models.py:31
msgid "过期"
msgstr ""

#: auth/models/auth_models.py:32
msgid "作废"
msgstr ""

#: auth/models/auth_models.py:161 auth/models/auth_models.py:183
#: auth/models/outer_models.py:202 auth/models/outer_models.py:239
#: auth/models/outer_models.py:429 auth/models/outer_models.py:576
#: auth/models/outer_models.py:605 auth/models/outer_models.py:634
msgid "项目ID"
msgstr ""

#: auth/models/auth_models.py:163 auth/models/outer_models.py:237
msgid "结果表标识"
msgstr ""

#: auth/models/auth_models.py:166 auth/models/outer_models.py:397
#: auth/models/ticket_models.py:88
msgid "创建人"
msgstr ""

#: auth/models/auth_models.py:167 auth/models/outer_models.py:398
#: auth/models/ticket_models.py:89
msgid "创建时间"
msgstr ""

#: auth/models/auth_models.py:169
msgid "作业描述"
msgstr ""

#: auth/models/auth_models.py:184 auth/models/auth_models.py:621
#: auth/models/outer_models.py:393 auth/views/auth_serializers.py:153
msgid "集群组ID"
msgstr ""

#: auth/models/auth_models.py:216
msgid "已过期"
msgstr ""

#: auth/models/auth_models.py:217
msgid "已禁用"
msgstr ""

#: auth/models/auth_models.py:218
msgid "已启用"
msgstr ""

#: auth/models/auth_models.py:219
msgid "已移除"
msgstr ""

#: auth/models/auth_models.py:620
msgid "原始数据ID"
msgstr ""

#: auth/models/auth_models.py:671
msgid "授权主体类型"
msgstr ""

#: auth/models/auth_models.py:672
msgid "授权主体"
msgstr ""

#: auth/models/auth_models.py:673 auth/models/outer_models.py:561
msgid "资源组ID"
msgstr ""

#: auth/models/auth_models.py:685 auth/views/auth_serializers.py:274
msgid "数据集类型"
msgstr ""

#: auth/models/auth_models.py:686 auth/views/auth_serializers.py:275
msgid "数据集ID"
msgstr ""

#: auth/models/auth_models.py:688 auth/views/auth_serializers.py:276
msgid "标记方式"
msgstr ""

#: auth/models/outer_models.py:201
msgid "作业名称"
msgstr ""

#: auth/models/outer_models.py:203 auth/models/outer_models.py:244
#: auth/models/outer_models.py:306 auth/models/outer_models.py:365
#: auth/models/outer_models.py:401 auth/models/outer_models.py:430
#: auth/models/outer_models.py:533
msgid "备注信息"
msgstr ""

#: auth/models/outer_models.py:241
msgid "结果表名"
msgstr ""

#: auth/models/outer_models.py:242
msgid "别名"
msgstr ""

#: auth/models/outer_models.py:243
msgid "结果表类型"
msgstr ""

#: auth/models/outer_models.py:245
msgid "计算类型"
msgstr ""

#: auth/models/outer_models.py:264
msgid "目前仅支持清洗表查询关联的原始数据"
msgstr ""

#: auth/models/outer_models.py:270
msgid "清洗表（{}）无法追溯到原始数据"
msgstr ""

#: auth/models/outer_models.py:281
msgid "血缘方向"
msgstr ""

#: auth/models/outer_models.py:282
msgid "数据源类型"
msgstr ""

#: auth/models/outer_models.py:283
msgid "数据ID"
msgstr ""

#: auth/models/outer_models.py:284
msgid "处理ID"
msgstr ""

#: auth/models/outer_models.py:394
msgid "集群组名称"
msgstr ""

#: auth/models/outer_models.py:395
msgid "集群中文名"
msgstr ""

#: auth/models/outer_models.py:396
msgid "集群可访问性"
msgstr ""

#: auth/models/outer_models.py:396
msgid "私有"
msgstr ""

#: auth/models/outer_models.py:396
msgid "公共"
msgstr ""

#: auth/models/outer_models.py:399
msgid "修改人"
msgstr ""

#: auth/models/outer_models.py:400
msgid "修改时间"
msgstr ""

#: auth/models/outer_models.py:428
msgid "图表名称"
msgstr ""

#: auth/models/outer_models.py:508
#, python-brace-format
msgid "[{id}] {app_code} 授权码"
msgstr ""

#: auth/models/outer_models.py:559
msgid "资源组英文标识"
msgstr ""

#: auth/models/outer_models.py:560
msgid "区域"
msgstr ""

#: auth/models/outer_models.py:572
msgid "模型ID"
msgstr ""

#: auth/models/outer_models.py:573
msgid "模型名称"
msgstr ""

#: auth/models/outer_models.py:574
msgid "模型别名"
msgstr ""

#: auth/models/outer_models.py:575 auth/models/outer_models.py:604
#: auth/models/outer_models.py:632
msgid "描述"
msgstr ""

#: auth/models/outer_models.py:578 auth/models/outer_models.py:607
msgid "是否激活"
msgstr ""

#: auth/models/outer_models.py:601
msgid "样本集ID"
msgstr ""

#: auth/models/outer_models.py:602
msgid "样本集名称"
msgstr ""

#: auth/models/outer_models.py:603
msgid "样本集别名"
msgstr ""

#: auth/models/outer_models.py:630
msgid "算法名称"
msgstr ""

#: auth/models/outer_models.py:631
msgid "算法别名"
msgstr ""

#: auth/models/tdw_models.py:15
msgid "用户名"
msgstr ""

#: auth/models/tdw_models.py:16
msgid "TDW用户名"
msgstr ""

#: auth/models/tdw_models.py:19
msgid "TDW密码"
msgstr ""

#: auth/models/ticket_models.py:62
msgid "系统自动审批"
msgstr ""

#: auth/models/ticket_models.py:72
msgid "创建人与审批人相同，自动过单"
msgstr ""

#: auth/models/ticket_models.py:90 auth/views/auth_serializers.py:183
msgid "申请理由"
msgstr ""

#: auth/models/ticket_models.py:91
msgid "当前申请状态"
msgstr ""

#: auth/models/ticket_models.py:92
msgid "结束时间"
msgstr ""

#: auth/models/ticket_models.py:93
msgid "处理总步骤数"
msgstr ""

#: auth/models/ticket_models.py:94
msgid "额外的内容"
msgstr ""

#: auth/models/ticket_models.py:95
msgid "当前处理步骤"
msgstr ""

#: auth/models/ticket_models.py:96
msgid "通用单据标识，目前仅用于通用单据"
msgstr ""

#: auth/models/ticket_models.py:255
msgid "可处理人"
msgstr ""

#: auth/models/ticket_models.py:256
msgid "处理人"
msgstr ""

#: auth/models/ticket_models.py:257
msgid "处理时间"
msgstr ""

#: auth/models/ticket_models.py:258
msgid "所属步骤"
msgstr ""

#: auth/models/ticket_models.py:259
msgid "处理信息"
msgstr ""

#: auth/models/ticket_models.py:260
msgid "处理状态"
msgstr ""

#: auth/models/ticket_models.py:365 auth/models/ticket_models.py:415
msgid "申请主体标识"
msgstr ""

#: auth/models/ticket_models.py:367 auth/models/ticket_models.py:417
msgid "申请主体"
msgstr ""

#: auth/models/ticket_models.py:368 auth/models/ticket_models.py:392
#: auth/models/ticket_models.py:418
msgid "申请操作方式"
msgstr ""

#: auth/models/ticket_models.py:369 auth/models/ticket_models.py:419
msgid "申请对象"
msgstr ""

#: auth/models/ticket_models.py:370 auth/models/ticket_models.py:420
msgid "资源key"
msgstr ""

#: auth/models/ticket_models.py:371 auth/models/ticket_models.py:421
msgid "资源值"
msgstr ""

#: auth/models/ticket_models.py:393
msgid "用户标识"
msgstr ""

#: auth/models/ticket_models.py:394 auth/views/auth_serializers.py:26
#: auth/views/auth_serializers.py:32 auth/views/auth_serializers.py:70
msgid "角色ID"
msgstr ""

#: auth/models/ticket_models.py:395
msgid "资源id"
msgstr ""

#: auth/permissions.py:30
msgid "单据不存在"
msgstr ""

#: auth/permissions.py:47
msgid "单据状态不存在"
msgstr ""

#: auth/services/token.py:106
#, python-format
msgid "APP(%s)的授权码"
msgstr ""

#: auth/services/token.py:138
msgid "暂不支持申请全部操作权限"
msgstr ""

#: auth/utils/serializer.py:37
msgid "scope参数校验失败"
msgstr ""

#: auth/views/auth_serializers.py:27 auth/views/auth_serializers.py:34
msgid "域ID"
msgstr ""

#: auth/views/auth_serializers.py:28 auth/views/auth_serializers.py:33
#: auth/views/auth_serializers.py:65 auth/views/auth_serializers.py:71
#: auth/views/auth_serializers.py:239 auth/views/auth_serializers.py:251
msgid "用户ID"
msgstr ""

#: auth/views/auth_serializers.py:33 auth/views/auth_serializers.py:65
#: auth/views/auth_serializers.py:71
msgid "用户列表"
msgstr ""

#: auth/views/auth_serializers.py:42
msgid "参数错误，角色不存在"
msgstr ""

#: auth/views/auth_serializers.py:45
#, python-format
msgid "%s该角色成员不能为空"
msgstr ""

#: auth/views/auth_serializers.py:51 auth/views/auth_serializers.py:73
msgid "角色用户列表"
msgstr ""

#: auth/views/auth_serializers.py:55 auth/views/auth_serializers.py:59
#: auth/views/auth_serializers.py:64
msgid "角色范围"
msgstr ""

#: auth/views/auth_serializers.py:60
msgid "权限对象"
msgstr ""

#: auth/views/auth_serializers.py:83 auth/views/auth_serializers.py:97
#, python-format
msgid "提交的角色不合法%s"
msgstr ""

#: auth/views/auth_serializers.py:102 auth/views/auth_serializers.py:112
msgid "对象类型"
msgstr ""

#: auth/views/auth_serializers.py:103
msgid "对象id"
msgstr ""

#: auth/views/auth_serializers.py:104
msgid "对象名搜索"
msgstr ""

#: auth/views/auth_serializers.py:105 auth/views/auth_serializers.py:123
msgid "是否返回显示名"
msgstr ""

#: auth/views/auth_serializers.py:106
msgid "页码"
msgstr ""

#: auth/views/auth_serializers.py:107
msgid "分页数量"
msgstr ""

#: auth/views/auth_serializers.py:108 auth/views/auth_serializers.py:118
msgid "是否用户模式"
msgstr ""

#: auth/views/auth_serializers.py:113 auth/views/auth_serializers.py:234
msgid "业务id"
msgstr ""

#: auth/views/auth_serializers.py:114 auth/views/auth_serializers.py:226
msgid "项目id"
msgstr ""

#: auth/views/auth_serializers.py:122 auth/views/auth_serializers.py:133
#: auth/views/auth_serializers.py:138 auth/views/auth_serializers.py:200
#: auth/views/auth_serializers.py:240 auth/views/auth_serializers.py:252
msgid "操作方式"
msgstr ""

#: auth/views/auth_serializers.py:124
msgid "是否管理员范围"
msgstr ""

#: auth/views/auth_serializers.py:125
msgid "是否考虑默认标准的TDW表"
msgstr ""

#: auth/views/auth_serializers.py:126
msgid "TDW表过滤参数"
msgstr ""

#: auth/views/auth_serializers.py:127
msgid "支持业务参数过滤"
msgstr ""

#: auth/views/auth_serializers.py:128 auth/views/auth_serializers.py:129
msgid "分页参数"
msgstr ""

#: auth/views/auth_serializers.py:134
msgid "维度字段"
msgstr ""

#: auth/views/auth_serializers.py:139 auth/views/auth_serializers.py:201
#: auth/views/auth_serializers.py:241
msgid "对象ID"
msgstr ""

#: auth/views/auth_serializers.py:140 auth/views/auth_serializers.py:246
msgid "是否展示鉴权详情"
msgstr ""

#: auth/views/auth_serializers.py:144
msgid "结果表ID"
msgstr ""

#: auth/views/auth_serializers.py:157
msgid "被删除的数据集"
msgstr ""

#: auth/views/auth_serializers.py:158
msgid "受影响的数据集"
msgstr ""

#: auth/views/auth_serializers.py:182
msgid "对象范围"
msgstr ""

#: auth/views/auth_serializers.py:184
msgid "过期时间"
msgstr ""

#: auth/views/auth_serializers.py:188
msgid "权限状态"
msgstr ""

#: auth/views/auth_serializers.py:189
msgid "展示权限对象属性"
msgstr ""

#: auth/views/auth_serializers.py:190
msgid "显示授权码组织结构"
msgstr ""

#: auth/views/auth_serializers.py:194
msgid "蓝鲸app"
msgstr ""

#: auth/views/auth_serializers.py:198
msgid "Data Token"
msgstr ""

#: auth/views/auth_serializers.py:199
msgid "APP CDDE"
msgstr ""

#: auth/views/auth_serializers.py:205
msgid "tdw用户名"
msgstr ""

#: auth/views/auth_serializers.py:206
msgid "tdw密码"
msgstr ""

#: auth/views/auth_serializers.py:210
msgid "集群id"
msgstr ""

#: auth/views/auth_serializers.py:214
msgid "库名"
msgstr ""

#: auth/views/auth_serializers.py:218
msgid "表名"
msgstr ""

#: auth/views/auth_serializers.py:222
msgid "TDW应用组"
msgstr ""

#: auth/views/auth_serializers.py:230
msgid "任务id"
msgstr ""

#: auth/views/auth_serializers.py:235
msgid "是否返回分页结构"
msgstr ""

#: auth/views/auth_serializers.py:245 auth/views/auth_serializers.py:256
msgid "鉴权列表"
msgstr ""

#: auth/views/auth_serializers.py:253
msgid "Dgraph语句变量名称"
msgstr ""

#: auth/views/auth_serializers.py:254
msgid "元数据对象类型"
msgstr ""

#: auth/views/auth_serializers.py:260
msgid "被交接人"
msgstr ""

#: auth/views/auth_serializers.py:283
msgid "当标记为 User 模式，敏感度字段必须传"
msgstr ""

#: auth/views/auth_serializers.py:286
msgid "暂不支持当前敏感度，请设置为其他敏感度"
msgstr ""

#: auth/views/user_views.py:154
msgid "目前 actoin_id 仅支持 {} 和 {} 维度"
msgstr ""

#: auth/views/user_views.py:413
msgid "操作方式 action_id 与对象类型 metadata_type 不一致"
msgstr ""
